.gitattributes

```diff
@@ -11,3 +11,5 @@
 /Tests              export-ignore
 /Resources/meta     export-ignore
 /Resources/doc      export-ignore
+/docker             export-ignore
+/.Makefile          export-ignore
```

Comment:
There's a typo here: it should be `/Makefile`, not `/.Makefile`

Reply:
ahahahah my bad :( sorry @garak